### DISCUSSION TITLE
How would i run a pyinstaller script from an exe compiled with pyinstaller?

### DISCUSSION THREAD

**Discussion by yTax** [pyinstaller/pyinstaller#6978] [Help]
Im currently working on a software that has two parts, the main file and the builder, the main file has a bunch of stuff that gets changed according to the users preferences when they open the builder, after thats done, the builder compiles the main file into an exe, thats where im having my problem when its supposed to run the part of the script from the builderthat makes pyinstaller compile main into an EXE, it works perfectly and compiles main into an EXE.
However if i compile the builder into an EXE, it will run the pyinstaller script but i stops after it tries to add paths to pythonpaths. 
![image](https://user-images.githubusercontent.com/65450189/180912901-6380478d-59f8-45f1-a4c6-2c412d3e0abd.png)
this image is the last thing that i see from pyinstaller before it goes back to the main menu of my builder.

**Comment by bwoodsend** [comment]
[sigh] A PyInstaller build is not, and will never be, capable of running PyInstaller.

**Reply by yTax** [reply]
oh that makes sense, sorry for bothering and wasting your time
